code/game/objects/items/storage/boxes/clothes_boxes.dm

```diff
@@ -211,3 +211,11 @@
 	new /obj/item/clothing/gloves/combat/floortile(src)
 	new /obj/item/clothing/shoes/jackboots/floortile(src)
 	new /obj/item/storage/backpack/floortile(src)
+
+/obj/item/storage/box/collar_bomb
+	name = "collar bomb box"
+	desc = "A small print on the back reads 'For research purposes only. Handle with care. In case of emergency, call the following number:'. the rest scratched out with a marker... welp."
```

Comment:
```suggestion
	desc = "A small print on the back reads 'For research purposes only. Handle with care. In case of emergency, call the following number:'... the rest is scratched out with a marker..."
```

clean it up and i think the marvel punchline, while funny, can probably go away